### PR TITLE
Add translator for Trains News Wire

### PR DESCRIPTION
**Pull request by mackensen**
New translator for the _Trains_ magazine News Wire feed at http://trn.trains.com/news/news-wire.

### CODE REVIEW

File: Trains News Wire.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
+	"translatorID": "3f4f0a1c-f542-4b44-8824-40ee1f921373",
+	"label": "Trains News Wire",
+	"creator": "Charles Fulton",
+	"target": "https?://trn.trains.com/news/news-wire/[0-9]+",
```
Comment (zuphilip) [inline Trains News Wire.js]: Add `^` at the start and escape the points  `.`.

File: Trains News Wire.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	return "blogPost";
+}
+
+function doWeb(doc, url) {
+	scrape(doc, url);
+}
+
+function scrape(doc, url) {
+	newItem = new Zotero.Item("blogPost");
+	newItem.url = doc.location.href;
```
Comment (zuphilip) [inline Trains News Wire.js]: Why not simply use the second parameter `url`?
(In some environments `doc.location` might be problematic...)

**Comment by AbeJellinek** [comment]
@mackensen, would it be possible to address these issues and lint?

**Comment by AbeJellinek** [comment]
Closed due to lack of activity. Embedded Metadata works well for this site.

**Pull request closed without merging**